.rubocop_todo.yml

```diff
@@ -17,6 +17,18 @@ Layout/EmptyLineAfterGuardClause:
   Exclude:
     - 'lib/common/models/collection.rb'
 
+# Offense count: 2
+# Cop supports --auto-correct.
+Layout/SpaceAroundMethodCallOperator:
+  Exclude:
+    - 'lib/saml/user_attributes/ssoe.rb'
+    - 'modules/appeals_api/spec/requests/v1/higher_level_reviews_controller_spec.rb'
+
+# Offense count: 1
+Lint/StructNewOverride:
+  Exclude:
+    - 'spec/request/authentication/standard_authentication.rb'
```

Comment:
So I commented out all the offenses and ran `make lint`.  The autocorrect fixed all but this one automatically.  Do you want to pull the other exceptions out of here?  I can push my fixes to your branch